ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSpec.java

```diff
@@ -419,4 +419,11 @@ public boolean isMigratingToExternalTable() {
   public void setMigratingToExternalTable() {
     isMigratingToExternalTable = true;
   }
+
+  public static void setReplId(Map<String, String> srcParameter, Map<String, String> destParameter) {
```

Comment:
This fix is irrelevant to the jira description. Can we make it another jira with scenario?